### ISSUE TITLE
[HW2] question about /articles page

### ISSUE THREAD

**Issue by jihoonrf** [swsnu/swppfall2019#64]
/articles 페이지에 대한 스펙을 보면 다음과 같은 조건이 있는데,

> Users should be able to clearly make out the whole list of articles including article id, (full) article title, and author name in this page.
> Upon clicking on the article title, users should be able to access the article's detail page (/articles/:id). These article titles must be button but not link.

이러한 article 리스트에 대한 id 속성값은 따로 주지 않아도 되는 것인가요?
id, title, name이 모두 표시되어야 하고 article title이 버튼이어야 한다는 조건은 있는데,
스펙에서 명시하고 있는 다른 컴포넌트들과는 달리 이 부분은 각각의 id값에 대한 조건이 따로 없는 점이 의아해서 질문드립니다.

**Comment by ktaebum** [comment]
Yes just show article id and Author name as plain text or whatever you want to
